rosa_cluster_admin/rosa_monitoring/rosa-managing-metrics.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="rosa-managing-metrics"]
+= Managing metrics
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-managing-metrics
+
+toc::[]
+
+This document provides an overview about how {product-title} metrics are collected, queried and visualized.
+
+// Understanding metrics
+include::modules/osd-monitoring-understanding-metrics.adoc[leveloffset=+1]
+
+// Setting up metrics collection for user-defined projects
+include::modules/osd-monitoring-setting-up-metrics-collection-for-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-deploying-a-sample-service.adoc[leveloffset=+2]
+include::modules/osd-monitoring-specifying-how-a-service-is-monitored.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See the link:https://github.com/openshift/prometheus-operator/blob/release-4.7/Documentation/api.md[Prometheus Operator API documentation] for more information on `ServiceMonitor` and `PodMonitor` resources.
```

Comment:
Is your support ok with GitHub links?

Reply:
That's a good question. @sferich888 do you know the policy for linking to Github is the same for ROSA and OSD as it is for OCP? I'm commenting this out until we know for sure.